### PR TITLE
Notifications: Add documentation for channels

### PR DESCRIPTION
**Pull request by AlexFalessi**

### CODE REVIEW

File: src/docs/user/references/Notification-service.adoc [hunk 1]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+[title="Channel subscription"]
+image::channel-sub.png[align="center"]
+
+==== Rules
+
+- Unlike other notifications, channel notifications are retroactive. It means that when a user subscribes to a channel that already had notifications
+, they will appear to the user.
+- Channel notifications cannot be deleted, the link:../admin/ProActiveAdminGuide.html#_notification_housekeeping[Housekeeping]
+takes care of it once their life span is exceeded.
+- Notification service administrators can subscribe to any channels. Other users must be part of at least one of the authorized group
```
Comment (Hiba-Alili) [inline src/docs/user/references/Notification-service.adoc]: one of the authorized group**s**

File: src/docs/user/references/Notification-service.adoc [hunk 2]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+defined in the channel in order to subscribe to a channel, unless all groups are authorized.
+
+==== Channel Creation
+
+All users are able to create channels for any groups. It is needed to provide a unique channel name and a list of groups
```
Comment (Hiba-Alili) [inline src/docs/user/references/Notification-service.adoc]: for any group.

File: src/docs/user/references/Notification-service.adoc [hunk 3]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+
+==== Channel Notification
+
+In order to send a channel notification a user must append the Channel_Notification task to a workflow.
+
+By default, If the user executing the workflow is a notification-service administrator, it will notify all channels. Otherwise
+it notifies all channels available to the user.
+
+In order to customize the notification, a user may update the task's variables to change the notification's message,
+it's severity and the notified channels provided as a comma separated list.
```
Comment (Hiba-Alili) [inline src/docs/user/references/Notification-service.adoc]: the notified channels **are** provided as a comma separated list.

**Review by Hiba-Alili — APPROVED** [review]

File: src/docs/user/references/Notification-service.adoc [hunk 4]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+, they will appear to the user.
+- Channel notifications cannot be deleted, the link:../admin/ProActiveAdminGuide.html#_notification_housekeeping[Housekeeping]
+takes care of it once their life span is exceeded.
+- Notification service administrators can subscribe to any channels. Other users must be part of at least one of the authorized groups
+defined in the channel in order to subscribe to a channel, unless all groups are authorized.
+
+==== Channel Creation
+
+All users are able to create channels for any group. It is needed to provide a unique channel name and a list of groups
+which will be able to subscribe the the channel.
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: to fix "which will be able to subscribe the the channel."

File: src/docs/user/references/Notification-service.adoc [hunk 5]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+
+It can either be all groups authorized, one or more groups of the current users, or any free input.
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: highlight all groups authorized if it is a key word

File: src/docs/user/references/Notification-service.adoc [hunk 6]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+
+[title="Channel creation"]
+image::create-channel.png[align="center"]
+
+==== Channel Management
+
+Channel management is where users can update channels by modifying a channel's name, add or remove authorized groups or delete it.
+
+A notification service administrator is able to update any channels, regardless if he is the creator of the channel or not.
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: is able to update any channels, whether he is the channel creator or not

File: src/docs/user/references/Notification-service.adoc [hunk 7]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+Other users can update only channels that they created.
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: have created

File: src/docs/user/references/Notification-service.adoc [hunk 8]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+
+To update a channel's name or authorized groups, simply make the update and click on the "Save" button.
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: make a single way of highlighting keywords like update, save, etc. Something like all bold or italic or whatever

File: src/docs/user/references/Notification-service.adoc [hunk 9]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+
+To delete a channel, click on the bin icon on the left of the channel definition.
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: same with bin icon

File: src/docs/user/references/Notification-service.adoc [hunk 10]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+
+[title="Channel creation"]
+image::channel-management.png[align="center"]
+
+==== Channel Notification
+
+In order to send a channel notification a user must append the Channel_Notification task to a workflow.
+
+By default, If the user executing the workflow is a notification-service administrator, it will notify all channels. Otherwise
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: if

File: src/docs/user/references/Notification-service.adoc [hunk 11]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+it notifies all channels available to the user.
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: will notify

File: src/docs/user/references/Notification-service.adoc [hunk 12]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+
+In order to customize the notification, a user may update the task's variables to change the notification's message,
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: to customize the notification message ?

File: src/docs/user/references/Notification-service.adoc [hunk 13]
```diff
@@ -116,6 +116,66 @@ and activate the notification method to start receiving notification by this mea
...
+it's severity and the notified channels which are provided as a comma separated list.
```
Comment (mboussaa) [inline src/docs/user/references/Notification-service.adoc]: it's => its

**Comment by fviale** [comment]
Retest this please

**Comment by fviale** [comment]
I mentioned it already, don't do branches of branches ! Please do this code change against master and not against updateNotifDoc

**Comment by AlexFalessi** [comment]
Replaced by PR https://github.com/ow2-proactive/documentation/pull/726

**Pull request closed without merging**